app/services/remotestorage.js

```diff
@@ -1,58 +1,78 @@
-import Ember from 'ember';
+import Service from '@ember/service';
+import RemoteStorage from 'npm:remotestoragejs';
+import Widget from 'npm:remotestorage-widget';
+import Shares from 'npm:remotestorage-module-shares';
 
-export default Ember.Service.extend({
+export default Service.extend({
 
+  rs: null, // remoteStorage instance
+  shares: null,
```

Comment:
Couldn't the shares rather be an `alias` of `rs.shares` instead?

Reply:
As `rs` is not an EmberObject, I don't think that works.